spec/javascripts/health-care/components/Nav.spec.jsx

```diff
@@ -17,12 +45,20 @@ describe('<Nav>', () => {
       expect(activeLinks[0].getAttribute('href')).to.equal(path);
     };
 
+    const expectTwoActiveLinks = (component, path) => {
+      history.replace(path);
+      const activeLinks = ReactTestUtils.scryRenderedDOMComponentsWithClass(component, 'usa-current');
+      expect(activeLinks).to.have.lengthOf(2);
```

Comment:
non-obvious regexps should always have a comment explaining them. 

In this case though, I feel like you actually should be assertting `activeLinks[1].getAttribute('href').startsWith(activeLinks[0].getAttribute('href'))` as the real thing we want to check is 2 active links where one is a parent path of another.